Engine/Results/BaseResultsHandler.cs

```diff
@@ -210,7 +215,7 @@ protected decimal GetNetReturn()
                     var trades = Algorithm.TradeBuilder.ClosedTrades;
 
                     statisticsResults = StatisticsBuilder.Generate(trades, profitLoss, equity, performance, benchmark,
-                        StartingPortfolioValue, Algorithm.Portfolio.TotalFees, totalTransactions);
+                        StartingPortfolioValue, Algorithm.Portfolio.TotalFees, totalTransactions, AssetsUnderManagementCapacityManager.AumCapacity);
```

Comment:
I think there is a small chance that `AssetsUnderManagementCapacityManager` is null if there is some error during init, we might add a check and use `0` if that's the case